custom-connectors/secplugs/apiProperties.json

```diff
@@ -0,0 +1,9 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#808080",
```

Comment:
@Ajaatsecplugs Please make sure that you are following instructions for [icon](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#icon) and `iconBrandColor` is actually the background color of your connector's icon.

Reply:
fixed